lib/computer.rb

```diff
@@ -47,12 +54,9 @@ def horizontal_generator(ship_length)
 
 
 ########## FIRE AT BOARD METHODS #################
-  def initial_targets
```

Comment:
I feel like this was a good refactor to eliminate the redundancy of valid targets. Having the valid targets in the initialize method allows us to take out that whole method and simplify firing at the board for each turn. Nice work!